src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/JoinableTaskExtensions.cs

```diff
@@ -13,6 +13,7 @@ public static class JoinableTaskExtensions
         /// <param name="joinableTask"> Joinable task to execute. </param>
         /// <param name="callerClassName"> Caller class name. </param>
         /// <param name="callerMemberName"> Caller member name. </param>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "VSTHRD110:Observe result of async calls", Justification = "https://github.com/NuGet/Client.Engineering/issues/956")]
```

Comment:
We use github issues as a TODO. Let's change the justification to a meaningful message instead.

Reply:
I'm not sure this is the right `FileAndForget`, but [VS's own implementation](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=%2Fsrc%2Fenv%2Fshell%2FPackageFramework%2FCurrent%2FShell%2FVsTaskLibraryHelper.cs&version=GBmain&line=723&lineEnd=754&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) ignores the threading analyzers. I think it's fundamental to the type of work the method is doing, and the API signature.

Reply:
Used their approach of adding a local var to store the JoinableTask and removed the suppression attribute.